tf/train.py

```diff
@@ -471,7 +470,9 @@ def read(x):
             output_types=(tf.string, tf.string, tf.string, tf.string,
                           tf.string))
         test_dataset = test_dataset.map(ChunkParser.parse_function)
-        test_dataset = test_dataset.prefetch(4)
+        if tfprocess.strategy is None: #Mirrored strategy appends prefetch itself with a value depending on number of replicas
```

Comment:
I think this is indented too much - it should run regardless of experimental_parser being true or false...

Reply:
fixed